source/docs/prismic/routable-types.md

```diff
@@ -69,7 +69,9 @@ Now you need to add your type to the GraphQL dispatcher query. For more info on
 
 ## Register the Prismic custom type as a routable type
 
-To make a custom Prismic type routable it must be registered with the Prismic module. First ensure that the `Prismic/Core` is added as a module's dependency then use `loaders.Prismic.registerRoutableType` to add the custom type as a routable type:
+To make a custom Prismic type routable it must be registered with the Prismic module. First ensure that the `Prismic/Core` is added as a module's dependency then;
+- use `loaders.Prismic.registerRoutableType` to add the custom type as a routable type
+- use `loaders.Prismic.registerPrismicRoute` to add allow the custom type to be handled by Prismic, see [Route Resolver](https://prismic.io/docs/core-concepts/link-resolver-route-resolver#route-resolver) for more information
```

Comment:
**issue:** it is not clear about the differences. Readers may think both calls are needed

**suggestion:** we should make each use case a bit clearer to better match the use case encountered by the developer

```suggestion
- either use `loaders.Prismic.registerRoutableType` to add the custom type as a routable type
- or use `loaders.Prismic.registerPrismicRoute` to inform Prismic about a custom URL pattern for a custom type, see [Route Resolver](https://prismic.io/docs/core-concepts/link-resolver-route-resolver#route-resolver) for more information
```

Reply:
Sorry yes, this is still a remenant of https://github.com/front-commerce/developers.front-commerce.com/pull/312 , I missed this, I will remove it for now, because I wan't to better document this under `Previews` where this becomes relevant.